tests/Browser/BrowserTestCase.php

```diff
@@ -53,7 +53,7 @@ protected function setUp(): void
     {
         // Don't accidentally run the migrations aganist the non-testing database. Ask me
         // how many times I've accidentally dropped my database...
-        if (env('DB_CONNECTION') !== 'testing') {
+        if (str_starts_with(env('DB_CONNECTION'), 'testing')) {
```

Comment:
Unneeded change.

Reply:
MySQL test connection is called `testing` and Postgres testing connection is called `testing-pgsql`, so I am pretty sure it is a necessary change so this bit of the code works for both the MySQL and PGSQL testing connections, and not just the MySQL one.